day-zero/live-templates.mdx

```diff
@@ -1,5 +1,209 @@
 ---
-title: 'Live Templates'
-description: 'Learn how platform leads can bundle blueprints, catalog properties, and plugins into templates to establish golden paths for new services.'
-icon: 'layer-group'
----
\ No newline at end of file
+title: "Live Templates"
+description: "Establish golden paths for your development team and maintain auto-updating standards with Amplication's Live Templates."
+icon: "layer-group"
+---
+
+<Frame>
+  <img src="/images/day-zero/live-service-templates.png" />
+</Frame>
+
+Amplication’s **Live Templates** let you define “golden paths” that standardize technology choices, security practices, and architectural patterns across your organization. By turning any resource—whether a **Blueprint** or an **Amplication Service**—into a published template, you ensure consistency from day one and greatly simplify long-term maintenance.
```

Comment:
a template includes a combination of plugins that are pre-configured. (Derick, that is the actual content of the template)